### PR TITLE
add skills and get skills 

### PR DESCRIPTION
**Pull request by JRakin**

### CODE REVIEW

File: components/ProfilePageComponent/SkillsSection.tsx [hunk 1]
```diff
@@ -39,6 +47,14 @@ const SkillsSection = () => {
         setIsOpen(false);
     }
 
+    useEffect(() => {
+        GetData('/api/skills?userId=' + _id).then((res: any) => {
+            console.log(res);
```
Comment (sifatul) [inline components/ProfilePageComponent/SkillsSection.tsx:52]: @JRakin bhai useCallback along with useEffect. you can look into experience section as an example

File: components/ProfilePageComponent/SkillsSection.tsx [hunk 2]
```diff
@@ -49,10 +65,23 @@ const SkillsSection = () => {
...
                         modalIsOpen={modalIsOpen}
                         closeModal={closeModal}
+                        skillTags={skillTags as any}
                     />
                 </div>
             </SkillsSectionHeaderContainer>
-            <SkillTags />
+            {skillTags && skillTags?.length ? (
+                <SkillTags tags={skillTags as any} />
+            ) : (
```
Comment (sifatul) [inline components/ProfilePageComponent/SkillsSection.tsx:74]: @JRakin please no tertiary condition inside mark up

**Comment by JRakin** [comment]
@sifatul vaiya please check

File: components/ProfilePageComponent/SkillsSection.tsx [hunk 3]
```diff
@@ -47,14 +47,18 @@ const SkillsSection = () => {
...
     }
 
-    useEffect(() => {
+    const getData = useCallback(() => {
         GetData('/api/skills?userId=' + _id).then((res: any) => {
             console.log(res);
             const { data = [], status } = res;
             if (status == 201) setSkillTags(data);
         });
     }, []);
```
Comment (sifatul) [inline components/ProfilePageComponent/SkillsSection.tsx:56]: @JRakin _id will be missing in first render. so add _id as dependency

**Review by sifatul — CHANGES_REQUESTED** [review]

File: components/ProfilePageComponent/SkillsSection.tsx (hunk 3 above)
Comment (JRakin) [reply components/ProfilePageComponent/SkillsSection.tsx]: @sifatul sorry my bad didn't notice fixed it

**Pull request merged**